CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # master
 
+# 2.13.0
+
+* Add the ability to disable the render monkey patch on < Rails 6.1 and include the `render_component` method in `ActionView` and `ActionController` which can be used for rendering components instead of `render`. Furthermore the method `render_component_to_string` is included in `ActionController` for rendering a component as string.
```

Comment:
```suggestion
* Add the ability to disable the render monkey patch with `config.view_component.render_monkey_patch_enabled`. In versions of Rails < 6.1, add `render_component` and `render_component_to_string` methods which can be used for rendering components instead of `render`.
```